app/scripts/modules/amazon/src/loadBalancer/configure/application/ALBListeners.tsx

```diff
@@ -820,13 +822,27 @@ const Action = (props: {
           </select>
         )}
         {props.action.type === 'redirect' && (
-          <button
-            className="btn btn-link no-padding"
-            type="button"
-            onClick={() => props.configureRedirect(props.action)}
-          >
-            Configure...
-          </button>
+          <dl className="dl-horizontal dl-narrow">
+            <dt>Host</dt>
+            <dd>{(props.action.redirectActionConfig || props.action.redirectConfig).host}</dd>
+            <dt>Path</dt>
+            <dd>{(props.action.redirectActionConfig || props.action.redirectConfig).path}</dd>
+            <dt>Port</dt>
+            <dd>{(props.action.redirectActionConfig || props.action.redirectConfig).port}</dd>
+            <dt>Protocol</dt>
+            <dd>{(props.action.redirectActionConfig || props.action.redirectConfig).protocol}</dd>
+            <dt>Status Code</dt>
+            <dd>{(props.action.redirectActionConfig || props.action.redirectConfig).statusCode}</dd>
```

Comment:
nit: Define a `const` to avoid repeating this code?

```js
const redirectConfig = props.action.redirectActionConfig || props.action.redirectConfig;

...

<dd>{redirectConfig.host}</dd>
```

Reply:
I was thinking about that too but ended up on the other side of the fence.